enlightenment_assistant_next-20160608.json

```diff
@@ -0,0 +1,17 @@
+{
+  "tags": [
+    "enlightenment_assistant_next"
+  ],
+  "area": [
+    {
+      "xpos": 457,
+      "ypos": 723,
+      "width": 111,
+      "height": 27,
+      "type": "match"
+    }
+  ],
+  "properties": [
+    "workaround"

```

Comment:
Fair point - the workaround was discussed with the Enlightenment maintainer of course... I'll file a bug for it and rename the needle